lib/libzfs/libzfs_util.c

```diff
@@ -600,8 +600,8 @@ zfs_setprop_error(libzfs_handle_t *hdl, zfs_prop_t prop, int err,
 			(void) zfs_error(hdl, EZFS_VOLTOOBIG, errbuf);
 			break;
 		}
+		zfsfallthrough;
```

Comment:
Should be outside of `#ifdef` I believe.

Reply:
Alas no, had to move it in. If `_ILP32` is not defined, it ends up being "empty", and we get
```
libzfs_util.c:604:3: error: expected expression
                zfsfallthrough;
                ^
../../lib/libspl/include/sys/feature_tests.h:35:25: note: expanded from macro
      'zfsfallthrough'
#define zfsfallthrough          __attribute__((__fallthrough__))
```

Reply:
Weird because GCC 11 works fine in this case. Anyways, it is not that important.
Enjoy OpenZFS on macOS!

```
#include <stdio.h>

int main() {
  int a = 1;
  switch (a) {
    case 1:
      printf("1\n");
  __attribute__((__fallthrough__));
    case 2:
  __attribute__((__fallthrough__));
    default:
      printf("d\n");
  }
}
```

Reply:
```
switch.c:8:3: warning: declaration does not declare anything
      [-Wmissing-declarations]
  __attribute__((__fallthrough__));
  ^
switch.c:10:3: error: expected expression
  __attribute__((__fallthrough__));
  ^
```
and make it
```
    case 2:
+    ;
  __attribute__((__fallthrough__));
```
Then just warnings.